### PR TITLE
Dain/noun 87 update collectiondao volume details

### PR DESCRIPTION
**Pull request by dblodorn**
Adding in dao specific auction volume stat with secondary sales sat omitting that amount.

Also did some layout touchup on the collection header/hero composition.

Dao Collection Page Header:
Screen Shot 2022-08-18 at 11 25 15 AM

Conventional Collection Page Header:
Screen Shot 2022-08-18 at 11 30 53 AM

### CODE REVIEW

File: components/MarketStats/CollectionStats.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { roundFourDecimals, roundTwoDecimals, numberFormatter } from '@shared'
+
+export function CollectionStats({ contractAddress }: { contractAddress: string }) {
+  const { aggregate } = useAggregate(contractAddress)
+
+  const volume = useMemo(
+    () =>
+      numberFormatter(
+        roundFourDecimals(
+          aggregate?.aggregateStat?.salesVolume?.chainTokenPrice as number
```
Comment (olegakbarov) [inline components/MarketStats/CollectionStats.tsx:13]: maybe `parseInt` or `Number(...)`?

**Review by olegakbarov — APPROVED** [review]
nits

File: compositions/Collections/CollectionHeader.tsx [hunk 2]
```diff
@@ -59,20 +55,40 @@ export function CollectionHeader({
...
+                collectionAddress={collection.address}
+                radius="round"
+                m="auto"
+                size="lg"
+              />
+            </Stack>
             <PageHeader
               headline={collection.name}
-              copy={`${aggregateStats.aggregateStat.nftCount} NFTs`}
+              copy={`${
```
Comment (bjfresh) [inline compositions/Collections/CollectionHeader.tsx:78]: Good place for a ??

```
{aggregate?.aggregateStat?.nftCount ?? '...'}
```

File: components/MarketStats/DaoStats.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  const auctionVolume = useMemo(() => {
+    if (data?.chainTokenPrice) {
+      return `${numberFormatter(roundFourDecimals(data?.chainTokenPrice as number))} ETH`
+    } else {
+      return '...'
+    }
+  }, [data, data?.chainTokenPrice])
+
+  const secondaryVolume = useMemo(() => {
```
Comment (bjfresh) [inline components/MarketStats/DaoStats.tsx:22]: With deep traversals like this I find it clearer to comprehend the logical flow by pulling the traversed result into a const with a name that's parsed easily at a glance, like:

```
const aggregateTokenPrice = aggregate?.aggregateStat?.salesVolume?.chainTokenPrice
```
so we then have

```if (data?.chainTokenPrice && aggregateTokenPrice)```

vs.

```if (data?.chainTokenPrice && aggregate?.aggregateStat?.salesVolume?.chainTokenPrice)```

Comment (dblodorn) [reply components/MarketStats/DaoStats.tsx:22]: Cool - giving this a shot.

**Pull request merged**